integration-tests/tests/analysis-context-bar.js

```diff
@@ -48,7 +48,8 @@ const testAnalysisContextBarFn = _.flow(
 
 const testAnalysisContextBar = {
   name: 'analysis-context-bar',
-  fn: testAnalysisContextBarFn
+  fn: testAnalysisContextBarFn,
+  timeout: 15 * 60 * 1000
```

Comment:
Waiting for Jupyter environment in Running status can take up to 10 min (L35). But, the default test timeout is set to [5 min](https://broadworkbench.atlassian.net/browse/QA-1872) which can cause test to fail if running time has exceeded 5 min.